Lib/test/test_base64.py

```diff
@@ -3,8 +3,16 @@
 import binascii
 import os
 from array import array
+from test.support import cpython_only
 from test.support import os_helper
 from test.support import script_helper
+from test.support.import_helper import ensure_lazy_imports
+
+
+class LazyImportTest(unittest.TestCase):
+    @cpython_only
+    def test_lazy_import(self):
+        ensure_lazy_imports("base64", {"re"})
```

Comment:
```suggestion
        ensure_lazy_imports("base64", {"re", "getopt"})
```
This is lazily imported in the CLI portion of the module